std-bits/base/src/main/java/org/enso/base/Text_Utils.java

```diff
@@ -160,6 +160,21 @@ public static int compare_normalized(String a, String b) {
     return Normalizer.compare(a, b, Normalizer.FOLD_CASE_DEFAULT);
   }
 
+  /**
+   * Compares {@code a} to {@code b} according to the lexicographical order, with optional Unicode
+   * normalization depending on the {@code isNormalized} parameter.
+   *
+   * @param a the left operand
+   * @param b the right operand
+   * @param isNormalized {@code true} if both input strings are normalized
+   * @return a negative value if {@code a} is before {@code b}, 0 if both values are equal and a
+   *     positive value if {@code a} is after {@code b}
+   */
+  public static int compare(String a, String b, boolean isNormalized) {
+    int options = isNormalized ? Normalizer.INPUT_IS_FCD : Normalizer.FOLD_CASE_DEFAULT;
```

Comment:
With default options, `Normalizer.compare` method checks if input strings are in FCD normal form every time. The `INPUT_IS_FCD` disables this check.

Reply:
Shouldn't this be
```suggestion
    int options = Normalizer.FOLD_CASE_DEFAULT | (isNormalized ? Normalizer.INPUT_IS_FCD : 0);
```

I think we should provide fold case default regardless of normalization status.

Reply:
I omitted default options because it is `0` and does not affect anything. Maybe it's worth including it if the value changes at some point. Updated